authors/apps/articles/views.py

```diff
@@ -119,3 +116,98 @@ def get_article(self, slug):
             return Article.objects.get(slug=slug)
         except Article.DoesNotExist:
             raise Http404
+
+
+class LikeView(APIView):
```

Comment:
This file is becoming too large so I would like for you to make a `views` folder into which you add groups of views. The file for this would then be something like LikesDislikesViews.py

Reply:
I have redistributed them, please checkout the changes